GVFS/GVFS.Notifications/VFSForGit.Mac/VFSForGit.UnitTests/ProductInfoFetcherTests.m

```diff
@@ -0,0 +1,111 @@
+#import <XCTest/XCTest.h>
+#import "ProductInfoFetcher.h"
+#import "MockNSTask.h"
+
+@interface ProductInfoFetcherTests : XCTestCase
+@end
+
+@implementation ProductInfoFetcherTests
+
+- (void)testProductInfoFetcherReturnsVFSForGitVersion
+{
+    NSString * const ExpectedVersion = @"GVFS Version 1.0.94768.54";
```

Comment:
Minor: The version format here (`"GVFS Version 1.0.94768.54"`) is different than the usual format of `gvfs --version`

```
~>gvfs --version
GVFS 0.6.19085.1
```